### PR TITLE
In-game input viewer

### PR DESCRIPTION
**Pull request by GSKirox**
This new cosmetic setting enables an in-game input viewer for the stick X/Y positions and all button presses except Start.
Dolphin screenshot : 
inputviewer

The icons used are the ones in the tutorial textboxes, which is why sadly Start is missing since they never needed one.
D-Pad inputs re-uses the C button ones, just colored differently.
The colors used for the A/B/C buttons will match their corresponding cosmetic.

The position used is almost certainly out of the CRT safe range, but this is an optional and not important rando information, and it's hard to find room on the screen without it being too distracting 😅

### CODE REVIEW

**Comment by r0bd0g** [comment]
I want to point out that the OoT Speedrun rules state:
"Adapters and input displays are not allowed to give you more information than looking at an unmodified controller (so no exact values, ess range indicators, etc)."

**Comment by GSKirox** [comment]
I'll bring it to the dev-race-mods channel !

**Comment by cjohnson57** [comment]
This is really cool! Hopefully we can find a solution where, if necessary, the viewer is disabled for race seeds and that's acceptable to the race mods.

File: SettingsList.py [hunk 1]
```diff
@@ -3778,6 +3778,16 @@ class SettingInfos:
         default        = False,
     )
 
+    input_viewer = Checkbutton(
+        gui_text       = 'Input viewer',
```
Comment (fenhl) [inline SettingsList.py]:
```suggestion
        gui_text       = 'Input Viewer',
```

**Comment by fenhl** [comment]
@GSKirox Could you rebase this please? I can give this a proper review then.

**Comment by GSKirox** [comment]
Rebased and pushed, sorry for the delay, i don't check github too often these days.

**Review by fenhl — CHANGES_REQUESTED** [review]
I've opened https://github.com/GSKirox/OoT-Randomizer/pull/5 with some code style fixes. This looks good otherwise.

**Review by fenhl — APPROVED** [review]

**Comment by GSKirox** [comment]
Thanks #dev-public-talk for making me check github sometimes...
Again sorry for the delay.

**Pull request merged**